plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/PipelineStore.java

```diff
@@ -69,6 +91,7 @@ public PipelineStore(Settings settings, ThreadPool threadPool, Environment envir
 
     @Override
     protected void doStart() {
+        client = injector.getInstance(Client.class);
```

Comment:
Simon suggested that we may be able to get it as a constructor argument? or would that create a cyclic dependency? maybe not if we get rid of the ClusterService dependency?